consensus/abci.go

```diff
@@ -94,127 +89,110 @@ func (abci *TendermintABCI) Info(req tmtAbciTypes.RequestInfo) tmtAbciTypes.Resp
 
 // SetOption sets a configuration option
 func (abci *TendermintABCI) SetOption(req tmtAbciTypes.RequestSetOption) tmtAbciTypes.ResponseSetOption {
-	abci.logger.Info("TendermintABCI::SetOption()")
 	return tmtAbciTypes.ResponseSetOption{Code: tmtAbciTypes.CodeTypeOK, Log: ""}
 }
 
 // InitChain initializes the validator set
 func (abci *TendermintABCI) InitChain(req tmtAbciTypes.RequestInitChain) tmtAbciTypes.ResponseInitChain {
-	abci.logger.Info("TendermintABCI::InitChain()")
```

Comment:
I would rather convert these output to `logger.Debug()` instead of removing them entirely as it brings value in case we need to some deep debugging in future

Reply:
I removed debug logs from empty methods, there is no point because there is nothing to debug.

Reply:
I think it is still useful to know which methods are being called in which order. I know it is not relevant as it is not code there, that is why I want to convert them to Debug(), so they will only appear on Debug lvl mode